core/modules/replica/testReplicaInfo.cc

```diff
@@ -0,0 +1,91 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2019 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+ /**
+  * @brief test ReplicaInfo
+  */
+
+// Third-party headers
+
+// LSST headers
+#include "lsst/log/Log.h"
+
+// Qserv headers
+#include "replica/ReplicaInfo.h"
+
+// Boost unit test header
+#define BOOST_TEST_MODULE ReplicaInfo
+#include "boost/test/included/unit_test.hpp"
+
+namespace test = boost::test_tools;
+using namespace lsst::qserv::replica;
+
+BOOST_AUTO_TEST_SUITE(Suite)
+
+BOOST_AUTO_TEST_CASE(ReplicaInfoTest) {
+
+    LOGS_INFO("ReplicaInfo test begins");
+
+    QservReplicaCollection one;
+    one.emplace_back(1U, "db1", 0);
+    one.emplace_back(1U, "db2", 0);   // only here
```

Comment:
"only here" not not very meaningful. Would it be worth expanding on this comment?

Reply:
Fixed. For the record, these comments are referring to those elements of the collections which are only present on one collection and not in an opposite collection. These elements are used for testing an algorithm which finds differences between collections.